src/OrchardCore.Modules/OrchardCore.Placements/Settings/PlacementContentTypePartDefinitionDriver.cs

```diff
@@ -17,46 +18,91 @@ public PlacementContentTypePartDefinitionDriver(IStringLocalizer<PlacementConten
+                        Description = S["{0} part in a {1} type", partDisplayName, displayName]
                     });
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
                     {
-                        ShapeType = partName,
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
                         ContentType = contentType,
                         DisplayType = "Detail",
-                        Description = S["Placement for the {0} part in a {1} type in detail views", partName, displayName]
+                        Description = S["{0} part in a {1} type in detail views", partDisplayName, displayName]
                     });
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
                     {
-                        ShapeType = partName,
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
                         ContentType = contentType,
                         DisplayType = "Summary",
-                        Description = S["Placement for the {0} part in a {1} type in summary views", partName, displayName]
+                        Description = S["{0} part in a {1} type in summary views", partDisplayName, displayName]
+                    });
+
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
+                    {
+                        Description = S["-"]
+                    });
+
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
+                    {
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
+                        ContentType = contentType,
+                        DisplayType = "DetailAdmin",
+                        Description = S["{0} part in a {1} type in admin detail views", partDisplayName, displayName]
                     });
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
+                    {
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
+                        ContentType = contentType,
+                        DisplayType = "SummaryAdmin",
+                        Description = S["{0} part in a {1} type in admin summary views", partDisplayName, displayName]
+                    });
+
+                model.PlacementSettingEntries.Add(
```

Comment:
What's the intent of this "empty" entry ?

Reply:
My preference is to just drop this divider.

Reply:
Ok, I'll remove it